oss-cloud-backend/handler.js

```diff
@@ -127,13 +127,24 @@ module.exports.updatePullRequests = async () => {
   return response;
 };
 
-module.exports.getContributions = async () => {
+module.exports.getContributions = async (event) => {
```

Comment:
Please add request body validation.

Reply:
I suggest to implement validation in the later stage of development.